types/node/http.d.ts

```diff
@@ -180,6 +180,7 @@ declare module "http" {
         httpVersion: string;
         httpVersionMajor: number;
         httpVersionMinor: number;
+        complete: boolean;
```

Comment:
Docs, tests, readonly?

Reply:
Links to docs & source are in the PR description. Per the source, it's not readonly.